drvregistry/drvregistry.go

```diff
@@ -157,20 +149,6 @@ func (r *DrvRegistry) IPAMDefaultAddressSpaces(name string) (string, string, err
 	return i.defaultLocalAddressSpace, i.defaultGlobalAddressSpace, nil
 }
 
-func (r *DrvRegistry) initIPAMs(lDs, gDs interface{}) error {
```

Comment:
Can we instead make this method exported so that in controller.go we just call it as

`drvRegistry.InitIPAMDrivers(nil, c.getStore(datastore.GlobalScope));`

we'd achieve the same thing I think.